server/app/views/components/ProgramQuestionBank.java

```diff
@@ -145,25 +148,49 @@ private FormTag questionBankPanel() {
                                     params.questionCreateRedirectUrl(),
                                     /* isPrimaryButton= */ false)))));
 
-    ImmutableList<QuestionDefinition> questions =
+    // Sort by last modified, since that's the default of the sort by dropdown
+    ImmutableList<QuestionDefinition> allQuestions =
         filterQuestions()
             .sorted(
                 Comparator.<QuestionDefinition, Instant>comparing(
                         qdef -> qdef.getLastModifiedTime().orElse(Instant.EPOCH))
                     .reversed()
                     .thenComparing(qdef -> qdef.getName().toLowerCase(Locale.ROOT)))
             .collect(ImmutableList.toImmutableList());
-
+    ImmutableList<QuestionDefinition> universalQuestions =
+        allQuestions.stream().filter(q -> q.isUniversal()).collect(ImmutableList.toImmutableList());
```

Comment:
suggestion (non-blocking):  It would be nice for the formatting on line 161 to match what's on lines 163-165.  Is it the formatter that's doing that?